ddd-koans/Value Object/Immutablity/src/Main.kt

```diff
@@ -0,0 +1 @@
+class TimeSignature(val numerator: Int, val denominator: Int)
```

Comment:
just thinking of making these properties `private` as well, what do you think?

Reply:
That is the thing I wanted to ask you, however, the question got lost sometime in the processes. I think that private would be better, but I am not sure about asserts in test cases, before equality lesson.

Reply:
I made them private, seems to work as all tests passed

Reply:
So should test for private properties be added?

Reply:
I think yes